src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1AssessmentDomainEventServiceTest.kt

```diff
@@ -387,14 +385,14 @@ class Cas1AssessmentDomainEventServiceTest {
     return assessment
   }
 
-  private fun assertStaffMemberDetailsMatch(staffMember: StaffMember?, staffDetails: StaffUserDetails?) = when {
+  private fun assertStaffMemberDetailsMatch(staffMember: StaffMember?, staffDetails: StaffDetail?) = when {
     staffMember == null -> staffDetails == null
     else ->
       staffDetails != null &&
-        staffMember.staffCode == staffDetails.staffCode &&
+        staffMember.staffCode == staffDetails.code &&
         staffMember.staffIdentifier == staffDetails.staffIdentifier &&
-        staffMember.forenames == staffDetails.staff.forenames &&
-        staffMember.surname == staffDetails.staff.surname &&
+        staffMember.forenames == staffDetails.name.forenames() &&
```

Comment:
Just checking this is a method call here `.forenames()` rather than accessing a property `.forenames`

Reply:
Great.. I wondered that..  Cheers.